lib/zoom.js

```diff
@@ -30,6 +28,14 @@ export default (container, dimensions, scales, configuration, data, callback) =>
         .attr('height', dimensions.outer_height)
         .attr('transform', `translate(${configuration.margin.left},35)`);
 
+    if (configuration.eventClick) {
+        zoomArea.on('click', () => circleHandler(zoomArea, configuration.eventClick));
```

Comment:
You have to save the result of the `on()` call to be able to deregister the handler, otherwise you're introducing a memory leak.